api/app/src/main/kotlin/packit/model/dto/SubmitRunInfo.kt

```diff
@@ -0,0 +1,8 @@
+package packit.model.dto
+
+data class SubmitRunInfo(
+    val name: String,
+    val branch: String,
+    val hash: String,
```

Comment:
Should branch and hash be nullable or do you always require both?

Reply:
both are required in the current implementation of the runner